build-package.sh

```diff
@@ -10,13 +17,34 @@ export TERMUX_SCRIPTDIR
 # Store pid of current process in a file for docker__run_docker_exec_trap
 source "$TERMUX_SCRIPTDIR/scripts/utils/docker/docker.sh"; docker__create_docker_exec_pid_file
 
+# Functions for working with packages
+source "$TERMUX_SCRIPTDIR/scripts/utils/package/package.sh"
 
 SOURCE_DATE_EPOCH=$(git log -1 --pretty=%ct 2>/dev/null || date "+%s")
 export SOURCE_DATE_EPOCH
 
 : "${TMPDIR:=/tmp}"
 export TMPDIR
 
+# file for storing compiled packages
+if [ "$TERMUX_BUILD_PACKAGE_CALL_DEPTH" = "0" ]; then
+	export TERMUX_BUILD_PACKAGE_CALL_BUILT_PACKAGES_LIST_FILE_PATH="${TMPDIR}/build-package-call-built-packages-list-$(date +"%Y-%m-%d-%H.%M.%S.")$((RANDOM%1000))"
+	echo -n " " > $TERMUX_BUILD_PACKAGE_CALL_BUILT_PACKAGES_LIST_FILE_PATH
+fi
+
+# check if the package is in the compiled list
+termux_check_package_in_built_packages_list() {
+	cat $TERMUX_BUILD_PACKAGE_CALL_BUILT_PACKAGES_LIST_FILE_PATH | grep " $1 " &>/dev/null
```

Comment:
This will not match first or last package in file. Use if condition provided. Also file existence should be checked.

Reply:
Use double quotes around paths.